internal/types/staticct/staticct.go

```diff
@@ -102,3 +103,45 @@ func (t *EntryBundle) UnmarshalText(raw []byte) error {
 	t.Entries = entries
 	return nil
 }
+
+// parseCTExtensions parses CTEXtensions into an index.
+// Code is inspired by https://github.com/FiloSottile/sunlight/blob/main/tile.go.
+func ParseCTExtensions(ext string) (uint64, error) {
+	extensionBytes, err := base64.StdEncoding.DecodeString(ext)
+	if err != nil {
+		return 0, fmt.Errorf("can't decode extensions: %v", err)
+	}
+	extensions := cryptobyte.String(extensionBytes)
+	var extensionType uint8
+	var extensionData cryptobyte.String
+	var leafIdx int64
```

Comment:
nit: this can't be -ve
```suggestion
	var leafIdx uint64
```